docs/metrics.md

```diff
@@ -144,8 +148,10 @@ Metrics below are captured within load balancer
 
 Metrics below are for invoker state as recorded within load balancer monitoring.
 
-* `openwhisk.counter.loadbalancer_invokerOffline_count` - Records the count of invokers considered offline based on health pings.
-* `openwhisk.counter.loadbalancer_invokerUnhealthy_count` - Records the count of invokers considered unhealthy based on health pings.
+* `openwhisk.counter.loadbalancer_totalHealthyInvoker<actionType>_count` - Records the count of managed invokers considered healthy based on health pings. actionType defines whether it is a managed or a blackbox action
```

Comment:
These two comments apply to this line and the following 3 as well.

* I understand that this is just documenting the current code so the question comes too late... why is the metric not per `<controllerId>`?
* I think the metric is not per `<actionType>` but per `<invokerType>`.

Reply:
also missing `.` at end of sentence here and below.
i'd use back ticks after "pings." to distinguish the value `<actionType>`  of if it's `<invokerType>`

Reply:
@sven-lange-last - I would move all the metric to be defined per invokerType. I was just trying to distinguish the names per metrics - probably not such a good idea 😉. Metric is again not defined per `controllerId`, as on the grafana query all the controller results are aggregated.